tests/spread/multipass/error-handling/snaps/exit1/snap/snapcraft.yaml

```diff
@@ -0,0 +1,18 @@
+name: exit1
+base: core18
```

Comment:
if using the set-base script in task.yaml, either remove this line here or add more logic to the script (the former would do for now)

Reply:
Line removed.